backtrace-library/src/main/java/backtraceio/library/BacktraceCredentials.java

```diff
@@ -76,4 +76,15 @@ public Uri getSubmissionUrl(){
         }
         return getServerUrl();
     }
+
+    public Uri getMinidumpSubmissionUrl() {
+        Uri backtraceJsonUri = getSubmissionUrl();
+        String jsonUrl = backtraceJsonUri.toString();
+        if(jsonUrl.contains("format=json")) {
+            jsonUrl = jsonUrl.replace("format=json", "format=minidump");
+        } else if (jsonUrl.contains(("/json"))) {
```

Comment:
you have unnecessarily one pair of parentheses here

Reply:
We will make a special case for invalid Backtrace URL - when URL doesn't ahve `/json` or `format=json` string. In this case we will return null and won't start NDK integration at all.